File: resource_management/resource_management.common/src/main/java/org/eclipse/slm/resource_management/common/remote_access/RemoteAccessRuntimeException.java
package org.eclipse.slm.resource_management.common.remote_access;

import org.springframework.http.HttpStatus;
import org.springframework.web.bind.annotation.ResponseStatus;

@ResponseStatus(value = HttpStatus.INTERNAL_SERVER_ERROR)
public class RemoteAccessRuntimeException extends RuntimeException{

    public RemoteAccessRuntimeException(String message){
        super(message);
    }

}

File: resource_management/resource_management.common/src/main/java/org/eclipse/slm/resource_management/common/remote_access/RemoteAccessRuntimeException.kt
package org.eclipse.slm.resource_management.common.remote_access

import org.springframework.http.HttpStatus
import org.springframework.web.bind.annotation.ResponseStatus

@ResponseStatus(value = HttpStatus.INTERNAL_SERVER_ERROR)
class RemoteAccessRuntimeException(message: String?, cause: Throwable?) : RuntimeException(message, cause)
